src/public/Interfaces.ts

```diff
@@ -1,54 +1,149 @@
+/**
+ * Interface for your Azure Function code. This function must be exported (via module.exports or exports) 
+ * and will execute when triggered. It is recommended that you declare this as an async function that 
+ * implicitly returns a Promise.
```

Comment:
"must be exported" contradicts the behavior when async. Or does this not apply here? Based on your second sentence, I think you should reword the first.

Also, "that implicitly returns a Promise" indicates that work needs to be done by the user, which I don't believe is the case. So maybe change to "async function, which will implicitly return a Promise"

Reply:
I think "an async function that implicitly returns" is correct

Reply:
^i see it now gotchu thank you!